Documentation/botr/clr-abi.md

```diff
@@ -149,7 +149,9 @@ For x86, all handlers are generated within the method body, typically in lexical
 
 JIT64 attempts to speed the normal control flow by 'inlining' a called finally along the 'normal' control flow (i.e., leaving a try body in a non-exceptional manner via C# fall-through). Because the VM semantics for non-rude Thread.Abort dictate that handlers will not be aborted, the JIT must mark these 'inlined' finally bodies. These show up as special entries at the end of the EH tables and are marked with `COR_ILEXCEPTION_CLAUSE_FINALLY | COR_ILEXCEPTION_CLAUSE_DUPLICATED`, and the try_start, try_end, and handler_start are all the same: the start of the cloned finally.
 
-JIT32 and RyuJIT currently do not implement finally cloning.
+There is now a prototype for cloned finallys in RyuJIT. It currently does not handle the thread abort case.
```

Comment:
Obviously, you'll want to be more verbose about the details here when the implementation gets finalized. E.g., that thread abort isn't handled since cloned finally clauses aren't reported.